up.go

```diff
@@ -7,12 +7,15 @@ import (
 	"fmt"
 	"sort"
 	"strings"
+
+	"go.uber.org/multierr"
```

Comment:
Will probably remove this package dep, just so happens I'm quite familiar with it.